ansible_runner/ansible_config.py

```diff
@@ -0,0 +1,74 @@
+############################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import logging
+from ansible_runner import base_config
+from ansible_runner.exceptions import ConfigurationError
+
+logger = logging.getLogger('ansible-runner')
+
+
+class AnsibleConfig(base_config.BaseConfig):
```

Comment:
I wonder if this should `AnsibleCfgConfig` ? Just to highlight that we are modifying ansible.cfg? I could go either way on this one.

Reply:
fixed